airflow/providers/exasol/hooks/exasol.py

```diff
@@ -216,22 +216,20 @@ def get_autocommit(self, conn) -> bool:
 
         :param conn: Connection to get autocommit setting from.
         :return: connection autocommit setting.
-        :rtype: bool
         """
         autocommit = conn.attr.get("autocommit")
         if autocommit is None:
             autocommit = super().get_autocommit(conn)
         return autocommit
 
     @staticmethod
-    def _serialize_cell(cell, conn=None) -> object:
+    def _serialize_cell(cell, conn=None) -> object:  # type: ignore[override]
```

Comment:
I think we can avoid this `ignore` by changing to return type to `Any`. Returning `object` is quite useless anyway.

Reply:
I like it, much cleaner thanks. Done for all 3 occurences of that, cf 409818b7450d57fb79a29a71c6be98a4b3adb6b8